src/renderer/store/selectors/channel.ts

```diff
@@ -181,20 +181,22 @@ export const inputLocked = createSelector(
   waggleSelectors.isConnected,
   isPublicChannel,
   (channelId, waggleContacts, waggle, publicChannel) => {
-    const contactsData: Array<{ publicKey: string }> = Object.values(waggleContacts)
+    const contactsData: Array<{ publicKey: string; nickname: string }> = Object.values(waggleContacts)
     const currentContactArray = contactsData.filter(item => {
-      return item.publicKey === channelId
+      return item.publicKey === channelId || item.nickname === channelId
     })
 
+    console.log('current contact')
+    console.log(currentContactArray[0])
     if (!waggle) {
       return INPUT_STATE.NOT_CONNECTED
     }
 
-    if (publicChannel || currentContactArray[0]) {
+    if (publicChannel || !currentContactArray[0]?.nickname?.startsWith('anon')) {
```

Comment:
Do we have anon users anymore? Is it possible? I don't think it is, since every user has a certificate.  

Though is the certificate included in every message? Is there some guarantee that all other users have seen the certificate before they see the message?

If it's not possible to see a user whose identity information you do not know, we should remove any code for displaying anon usernames. 

If there's something else we need to do to achieve this, let's create an issue and discuss.

Reply:
But you may have 'anon' users in your contacts, and that part is just filtering them to not appear. It's for returning users that may have old structures in store.